specutils/tests/conftest.py

```diff
@@ -1,11 +1,33 @@
 import pytest
 import requests
+import os
 
+import tempfile
 
-REMOTE_DATA_NAMES = [
 
-]
+REMOTE_DATA_IDS = {
+    'cos_fuv': ('1481119', 'COS_FUV.fits')
+}
 
+REMOTE_DATA_NAMES = {
+    'COS_FUV.fits': '1481119'
+}
 
-@pytest.fixture(scope='module', params=REMOTE_DATA_NAMES)
+ACCESS_TOKEN = os.environ.get("ACCESS_TOKEN")
```

Comment:
Sorry for the nit, but can I suggest maybe `ZENODO_ACCESS_TOKEN` for the environment variable just to avoid an accidental collision.